packages/web3torrent/tracker/index.js

```diff
@@ -0,0 +1,75 @@
+const Server = require('bittorrent-tracker').Server;
```

Comment:
If we were to deploy this tracker so that it was publicly accessible, would it track non-web3torrent torrents? If so, is there any easy way for it to only track web3torrents?

Reply:
Yeah, I read about [this](https://github.com/webtorrent/bittorrent-tracker#server) on the project, we could add some sort of secret or something to the clients, and pass somehow that secret to the tracker so that it only serves web3torrent clients.

Reply:
Yeah I think anything like that would be good -- it probably doesn't need to be super secure. 

We could filter on the `pseAccount` being a valid Ethereum address https://ethereum.stackexchange.com/questions/1374/how-can-i-check-if-an-ethereum-address-is-valid

Reply:
uhh, that's cool. Maybe I can add it in another PR? I think it involves changing the internal peerId (which is supposed to be read-only) after the client was instantiated. And we get the pseAccount after the web3torrent is created.

Reply:
Yeah separate PR is cool. 

 I'm not sure what ends up in the `params` of the server `filter` function: anything that picks out web3Torrent uniquely would work, I guess.